driver/razerkbd_driver.c

```diff
@@ -2407,6 +2443,9 @@ static void razer_kbd_disconnect(struct hid_device *hdev)
             break;
         }
     } else if(intf->cur_altsetting->desc.bInterfaceProtocol == USB_INTERFACE_PROTOCOL_KEYBOARD) {
+        // Translations
+        device_remove_file(&hdev->dev, &dev_attr_button_translations);
+
```

Comment:
Is there a `razer_cleanup_translations` call missing in this file?

Reply:
You're right i forgot to move it when after develop in mouse driver ;)